tools.json

```diff
@@ -7,15 +7,18 @@
     "git": "https://github.com/pricter/p5pj",
     "supportedPlatforms": [
       "Windows",
-      "Linux"
+      "Linux",
+      "Mac"
     ],
     "run": {
       "Windows": "python C:/toolbox/p5pj/p5pj-windows.py",
-      "Linux": "python3 ~/.toolboxlauncher/p5pj/p5pj-linux.py"
+      "Linux": "python3 ~/.toolboxlauncher/p5pj/p5pj-linux.py",
+      "Mac": "python3 ~/.toolboxlauncher/p5pj/p5pj-linux.py"
```

Comment:
for now I am just launching the linux python script on mac OS. This should not be permanent, but since it is python it runs fine. Eventually if a person develops a tool, each OS should be  tested and supplied a path that will definitely work.